server/src/it/scala/org/ensime/intg/BasicWorkflow.scala

```diff
@@ -164,14 +163,14 @@ class BasicWorkflow extends WordSpec with Matchers
       val insPacByPathResOpt = project.rpcInspectPackageByPath("org.example")
       insPacByPathResOpt match {
         case Some(PackageInfo("example", "org.example", List(
-          BasicTypeInfo("Bloo", _: Int, 'class, "org.example.Bloo", List(), List(), Some(EmptySourcePosition()), None),
-          BasicTypeInfo("Bloo$", _: Int, 'object, "org.example.Bloo$", List(), List(), Some(EmptySourcePosition()), None),
-          BasicTypeInfo("CaseClassWithCamelCaseName", _: Int, 'class, "org.example.CaseClassWithCamelCaseName", List(), List(), Some(EmptySourcePosition()), None),
-          BasicTypeInfo("CaseClassWithCamelCaseName$", _: Int, 'object, "org.example.CaseClassWithCamelCaseName$", List(), List(), Some(EmptySourcePosition()), None),
-          BasicTypeInfo("Foo", _: Int, 'class, "org.example.Foo", List(), List(), Some(EmptySourcePosition()), None),
-          BasicTypeInfo("Foo$", _: Int, 'object, "org.example.Foo$", List(), List(), Some(EmptySourcePosition()), None),
-          BasicTypeInfo("package$", _: Int, 'object, "org.example.package$", List(), List(), Some(EmptySourcePosition()), None),
-          BasicTypeInfo("package$", _: Int, 'object, "org.example.package$", List(), List(), Some(EmptySourcePosition()), None)))) =>
+          BasicTypeInfo("Bloo", _: Int, 'class, "org.example.Bloo", List(), List(), Some(_), None),
+          BasicTypeInfo("Bloo$", _: Int, 'object, "org.example.Bloo$", List(), List(), Some(_), None),
+          BasicTypeInfo("CaseClassWithCamelCaseName", _: Int, 'class, "org.example.CaseClassWithCamelCaseName", List(), List(), Some(_), None),
+          BasicTypeInfo("CaseClassWithCamelCaseName$", _: Int, 'object, "org.example.CaseClassWithCamelCaseName$", List(), List(), Some(_), None),
+          BasicTypeInfo("Foo", _: Int, 'class, "org.example.Foo", List(), List(), None, None),
+          BasicTypeInfo("Foo$", _: Int, 'object, "org.example.Foo$", List(), List(), Some(_), None),
+          BasicTypeInfo("package$", _: Int, 'object, "org.example.package$", List(), List(), None, None),
+          BasicTypeInfo("package$", _: Int, 'object, "org.example.package$", List(), List(), None, None)))) =>
```

Comment:
Interesting (as weith the old code - two "package$" items - they have different type ids.